### PR TITLE
Add a proto PID response for TPC signals

### PR DESCRIPTION
**Pull request by njacazio**
- add base class for parametrization
- add table for TPC PID quantities
- add TPC PID task

### CODE REVIEW

File: Analysis/Tasks/spectraTPC.cxx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  DOTH2F(hnsigmaKa, ";#it{p} (GeV/#it{c});TPC N_{sigma K};Tracks", 100, 0, 5, 100, -10, 10);
+  DOTH2F(hnsigmaPr, ";#it{p} (GeV/#it{c});TPC N_{sigma p};Tracks", 100, 0, 5, 100, -10, 10);
+  DOTH2F(hnsigmaDe, ";#it{p} (GeV/#it{c});TPC N_{sigma d};Tracks", 100, 0, 5, 100, -10, 10);
+  DOTH2F(hnsigmaTr, ";#it{p} (GeV/#it{c});TPC N_{sigma t};Tracks", 100, 0, 5, 100, -10, 10);
+  DOTH2F(hnsigmaHe, ";#it{p} (GeV/#it{c});TPC N_{sigma ^{3}He};Tracks", 100, 0, 5, 100, -10, 10);
+  DOTH2F(hnsigmaAl, ";#it{p} (GeV/#it{c});TPC N_{sigma #alpha};Tracks", 100, 0, 5, 100, -10, 10);
+
+  void process(aod::Collision const& collision, soa::Join<aod::Tracks, aod::TracksExtra, aod::pidRespTPC> const& tracks)
+  {
+    for (auto i : tracks) {
```
Comment (wiechula) [inline Analysis/Tasks/spectraTPC.cxx:48]: Shouldn't this be `const auto&` to avoid copy?

File: Analysis/Tasks/spectraTPC.cxx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      // Track selection
+      UChar_t clustermap = i.itsClusterMap();
```
Comment (wiechula) [inline Analysis/Tasks/spectraTPC.cxx:50]: `const`?

File: Analysis/Tasks/spectraTPC.cxx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      bool issel = (i.tpcNClsFindable() > 70) && (i.flags() & 0x4) && (TESTBIT(clustermap, 0) || TESTBIT(clustermap, 1));
```
Comment (wiechula) [inline Analysis/Tasks/spectraTPC.cxx:51]: `const`?

File: Analysis/DataModel/include/PID/ParamBase.h [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      Set(i, value[i]);
+  }
+  /// Getter for parametrization parameters
+  T Get(unsigned int param_index) const { return param_index < size ? mPar[param_index] : -999.f; }
+  /// Getter for parametrization parameters
+  const T* Get() const { return mPar; }
+
+ private:
+  /// parameters
+  T mPar[size];
```
Comment (wiechula) [inline Analysis/DataModel/include/PID/ParamBase.h:51]: `std::array`?

File: Analysis/DataModel/include/PID/PIDResponse.h [hunk 5]
```diff
@@ -147,4 +176,39 @@ struct pidTOFTask {
...
+  Produces<aod::pidRespTPC> tpcpid;
+
+  void process(aod::Collision const& collision, soa::Join<aod::Tracks, aod::TracksExtra> const& tracks)
+  {
+    tpc::Response resp = tpc::Response();
+    float bbparams[5] = {0.0320981, 19.9768, 2.52666e-16, 2.72123, 6.08092};
+    resp.mParam.mBetheBloch.mParameters.Set(bbparams);
+    float resoparams[2] = {0.07, 0.0};
+    resp.mParam.mRelResolution.mParameters.Set(resoparams);
+    for (auto i : tracks) {
```
Comment (wiechula) [inline Analysis/DataModel/include/PID/PIDResponse.h:189]: Shouldn't this be `const auto&` to avoid copy?

**Comment by njacazio** [comment]
Can we merge this?

**Comment by njacazio** [comment]
@ktf it should be ready for merge

**Comment by ktf** [comment]
In general, can you please squash commits when you are ready for something to be merged? I can do it on my side, but then the commit message looks weird...

**Pull request merged**